src/www/services_rfc2136_edit.php

```diff
@@ -58,6 +58,9 @@
     $pconfig['usetcp'] = isset($a_rfc2136[$id]['usetcp']);
     $pconfig['usepublicip'] = isset($a_rfc2136[$id]['usepublicip']);
 
+    $pconfig['recordtype'] = $a_rfc2136[$id]['recordtype'];
+    if (!$pconfig['recordtype']) $pconfig['recordtype'] = "both";
```

Comment:
Most of it is in the handbook: https://docs.opnsense.org/development/workflow.html

But it needs some updates on the refined tools :)